CONTRIBUTING.md

```diff
@@ -68,19 +68,19 @@ A pull request doesn't have to represent finished work. It's usually better to o
 
 ### Pro Tips to follow
 
-- **Don't mind about the style** because we use [StandardJS](https://github.com/standard/standard), [ESLint](https://github.com/eslint/eslint) and [Prettier](https://github.com/prettier/prettier). Use `npm run lint` command.
-- **Don't change the markdown files**, because the README is generated (it isn't hand written) and the API section is from JSDoc code comments. Let this step to us when _and if_ pull request is merged.
-- **Dont't comment tests**, instead use `test.skip`. They'll still be shown in the output, but are never run.
+- **Don't worry about the style** because we use [StandardJS](https://github.com/standard/standard), [ESLint](https://github.com/eslint/eslint) and [Prettier](https://github.com/prettier/prettier). Use the `npm run lint` command.
```

Comment:
yea... i just always worry and try to fit in the github size ;d because i don't like whole new line for one word, it's not eye handy.